GitCommands/Settings/AppSettings.cs

```diff
@@ -1839,6 +1839,12 @@ public static DiffListSortType DiffListSorting
             set => SetEnum("DiffListSortType", value);
         }
 
+        public static bool DontConfirmBranchCheckout
+        {
+            get => GetBool("DontConfirmBranchCheckout", true);
+            set => SetBool("DontConfirmBranchCheckout", value);
+        }
```

Comment:
I _think_ all other similar settings are grouped together. If so, please put the new settings near those.

Reply:
For new settings, please use:
```suggestion
        public static readonly ISetting<bool> DontConfirmBranchCheckout= Setting.Create(ConfirmationsSettingsPath , nameof(DontConfirmBranchCheckout), true);
```

Needs to be added to the top of the file:
```
        private static readonly SettingsPath ConfirmationsSettingsPath = new AppSettingsPath("Confirmations");
```

Reply:
Some of the settings are grouped together, but not all of them. After applying the suggested code changes, I put the new setting near some of the branch settings.